.gitignore

```diff
@@ -14,4 +14,5 @@ test-output/
 /bin/
 *.log
 config/
-rawlsdb/
\ No newline at end of file
+rawlsdb/
+*.pid
```

Comment:
@jmthibault79 The docker run script creates a `.pid` file while running, to facilitate cleanup in case it's interrupted. This is already ignored in the other repos.